src/release-notes/v2021.12.08-0.md

```diff
@@ -0,0 +1,18 @@
+---
+date: "2021-12-08"
+version: "v2021.12.08-0"
+weight: 202112080
+---
+
+### <span class="label label-green">New Features</span>
```

Comment:
I think previously we would call these `Improvements`.

Reply:
I see that in some of the prior release notes have that Improvements section.  How does that different from New Features?